test/local.js

```diff
@@ -66,6 +66,17 @@ describe('UploadFS Local', function () {
     });
   });
 
+  it('streamOut should work for local filesystem', async function() {
+    const input = uploadfs.streamOut('/test_copy.txt');
+    const chunks = [];
+    for await (let chunk of input) {
```

Comment:
I keep forgetting this really convenient idiom exists for reading from streams in Node, no need to set up a billion event handlers.